### PR TITLE
Feature/x y

### PR DESCRIPTION
**Pull request by pengin7384**
Done
- Add X, Y

### CODE REVIEW

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: ain_benchmark.js [hunk 1]
```diff
@@ -243,7 +256,8 @@ function printTestResult(testList) {
     }
     console.log();
   }
-  console.log(`Total TPS: ${totalTps.toFixed(5)}`);
+  console.log(`Total TPS (X): ${totalTps.toFixed(5)}`);
+  console.log(`Number of paths in state database (Y): ${numberOfPaths}`);
```
Comment (platfowner) [inline ain_benchmark.js:260]: What about more direct message? 'Number of shards (sharding paths) (Y): ...'

**Review by platfowner — APPROVED** [review]
LGTM 
with some minor comments.

File: ain_benchmark.js (hunk 1 above)
Comment (pengin7384) [reply ain_benchmark.js]: Done! Thanks

**Pull request merged**